packages/service-library/src/servicelib/fastapi/prometheus_instrumentation.py

```diff
@@ -54,9 +54,23 @@ def _on_shutdown() -> None:
     return get_prometheus_instrumentator(app)
 
 
-async def lifespan_prometheus_instrumentation(app: FastAPI) -> AsyncIterator[State]:
+_PROMETHEUS_INSTRUMENTATION_ENABLED: Final[str] = "prometheus_instrumentation_enabled"
+
+
+def get_prometheus_instrumentationmain_main_lifespan(*, enabled: bool) -> State:
+    return {_PROMETHEUS_INSTRUMENTATION_ENABLED: enabled}
```

Comment:
same here

Reply:
@pcrespov was there not some common naming as well for these functions?

Reply:
we now have a better naming patter